### PR TITLE
Fixed minor typo in docs/1.0/data-modelling 

### PR DESCRIPTION
**Pull request by evanrs**

### CODE REVIEW

File: docs/1.0/04-Reference/04-Database/01-Data-Modelling-(SDL).md [hunk 1]
```diff
@@ -426,7 +426,7 @@ type Story @model @rename(oldName: "Post") {
 
 The temporary directive `@migrationValue(value: String!)` is used to migrate the value of a scalar field. When changing an optional field to a requried field, it's necessary to also use this directive.
 
-## Naming onventions
+## Naming Conventions
```
Comment (kbrandwijk) [inline docs/1.0/04-Reference/04-Database/01-Data-Modelling-(SDL).md]: The rest of the headers use single capital, so `Naming conventions`.

Comment (evanrs) [reply docs/1.0/04-Reference/04-Database/01-Data-Modelling-(SDL).md]: Ah, my mistake, thank you for catching that.

Comment (evanrs) [reply docs/1.0/04-Reference/04-Database/01-Data-Modelling-(SDL).md]: Fixed.

**Comment by nikolasburk** [comment]
Thanks a lot for catching this @evanrs and for the review @kbrandwijk  🙌

**Pull request merged**